packages/yoga/src/Icon/Icon.jsx

```diff
@@ -49,13 +58,38 @@ Icon.propTypes = {
   /** Vertical size of the SVG. Use it as one of
    * theme.spacing tokens (xxsmall, small, medium...) */
   height: oneOf([...commonSizes, instanceOf(Number)]),
+  /** Size for vertical and horizontal of the SVG.
+   * Use it as one of theme.spacing tokens (xxsmall, small, medium...) */
+  size: (props, propName, componentName) => {
+    const { size, width, height } = props;
+
+    if (width || height) {
+      return new Error(
```

Comment:
Sorry, I don't really know how this prop type validation really works, but we usually `throw` errors, should we change here? Do we really need to return?

Reply:
If someone uses `size` they don't need to use `width` or `height`. I think we just need to add `size` as a conditional too.

Talking about returning new Error instead of throwing it, we're just following the guidelines from [`prop-types` API](https://reactjs.org/docs/typechecking-with-proptypes.html):

```jsx
  // You can also specify a custom validator. It should return an Error
  // object if the validation fails. Don't `console.warn` or throw, as this
  // won't work inside `oneOfType`.
  customProp: function(props, propName, componentName) {
    if (!/matchme/.test(props[propName])) {
      return new Error(
        'Invalid prop `' + propName + '` supplied to' +
        ' `' + componentName + '`. Validation failed.'
      );
    }
  },
```

Reply:
In that case, if the developer don't pass size and width && height are empty, why we doesn't set a defaultValue following size guide to device? (Web, Native and etc).

Reply:
If you don't provide any size prop (`width`, `height`, or `size`) the Icon will render as 12x12 px, as it is the default value for all Icon on [@gympass/yoga-icons](https://github.com/Gympass/yoga/blob/master/packages/icons/src/svg/accessibility.svg?short_path=7d5248a#L1). But, sure, we can provide a default size if it is a thing.

I also think we should discuss this subject despite this pull request and add this potential default behavior later. Is it ok for you, @willianpossa ?

Reply:
Of course @ggdaltoso!

I believe this behavior is a feature that we can implement later (Get size according device used)